openfe/protocols/openmm_afe/base.py

```diff
@@ -234,7 +235,7 @@ def _handle_settings(self):
           * solvation_settings : SolvationSettings
           * alchemical_settings : AlchemicalSettings
           * lambda_settings : LambdaSettings
-          * sampler_settings : AlchemicalSamplerSettings
+          * sampler_settings : MultiStateSimulationSettings
```

Comment:
Is it necessary to have `sampler_settings` here of could the `simulations_settings` just be the `MultiStateSimulationsSettings`?